tests/x11/evolution.pm

```diff
@@ -17,10 +17,12 @@ use testapi;
 
 sub run() {
     x11_start_program("evolution");
+    if ((get_var('FLAVOR') =~ /Gnome-Live/) && check_screen("evolution-preview-release", 20)) {
+        send_key "alt-o";   # OK
+    }
     if (check_screen "evolution-default-client-ask", 20) {
         assert_and_click "evolution-default-client-agree";
     }
-    assert_screen 'test-evolution-1', 30;
```

Comment:
hmm. that seems weird and unintational.. this is the main evo screen to be found; must be a mistake